compose/container.py

```diff
@@ -212,6 +212,21 @@ def restart(self, **options):
     def remove(self, **options):
         return self.client.remove_container(self.id, **options)
 
+    def create_exec(self, command, **options):
+        privileged = options.get("privileged")
+        user = options.get("user")
+        tty = options.get("tty")
+        return self.client.exec_create(
+            self.id, command, privileged=privileged, tty=tty, stdin=tty, user=user)
+
+    def start_exec(self, exec_id, **options):
+        detach = options.get("detach")
+        tty = options.get("tty")
+        return self.client.exec_start(exec_id, detach=detach, tty=tty)
+
+    def get_exec_exit_code(self, exec_id):
+        return self.client.exec_inspect(exec_id).get("ExitCode")
```

Comment:
Also a minor thing: this isn't a container-related method, so I feel like we could just delete it and move this line to `exec_command()` in `main.py`.